modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts

```diff
@@ -59,6 +80,42 @@ const getEndPointNode = (editor: Editor, rng: Range, start: Boolean, root: Node)
     container = container.nextSibling;
   }
 
+  if (start && NodeType.isTextNode(container)) {
+    if (Unicode.isZwsp(container.textContent as string)) {
+      if (container.previousSibling !== null && isInline(SugarElement.fromDom(container.previousSibling))) {
+        container = container.previousSibling.firstChild || container.previousSibling;
+      } else if (container.nextSibling !== null && isInline(SugarElement.fromDom(container.nextSibling))) {
+        container = container.nextSibling.firstChild || container.nextSibling;
+      }
+    }
+    if (!Unicode.isZwsp(container.textContent as string)) {
+      if ( container.parentNode !== null && isInline(SugarElement.fromDom(container.parentNode))) {
```

Comment:
It's an unnecessary space right after the round bracket, I believe. (Why eslint hasn't figured this out?)

Reply:
Good catch! fixed 👍